### DISCUSSION TITLE
Are there plans to support the Airflow scheduling engine?

### DISCUSSION THREAD

**Discussion by hectorhe001** [apache/seatunnel#4535] [General]
There are also a lot of users using Airflow now, especially for users in Europe and the US, and it would be great if the community could support it.

**Comment by dijiekstra** [comment]
Ok, we will support for airflow or others as soon as possible

hectorhe001 ***@***.***> 于2023年4月10日周一 17:27写道：

**Reply by hectorhe001** [reply]
Thanks, look forward to using it soon!

**Comment by liunaijie** [comment]
is there any update? And for this feature, i think it should be implement in airflow, not in seatunnel.
